13-TeV-examples/uproot_python/HZZAnalysis.ipynb

```diff
@@ -144,7 +146,8 @@
     "\n",
     "We're going to be using a number of tools to help us:\n",
     "* uproot: lets us read .root files typically used in particle physics into data formats used in python\n",
-    "* pandas: lets us store data as dataframes, a format widely used in python\n",
+    "* awkward: lets us store data as awkward arrays, a format that generalizes numpy to nested data with possibly variable length lists\n",
```

Comment:
awkard might be a better choice (over pandas) here since it simplifies handling of variable length lists (here the leptons)